bin/deploy

```diff
@@ -26,3 +26,9 @@ cf zero-downtime-push $CF_APP -f "$CF_APP"_manifest.yml
 curl -X POST \
 --data-urlencode 'payload={"text": "'"Deployment finished for $CF_APP"'", "channel": "'"#$SLACK_CHANNEL"'", "username": "cf-deploy", "icon_emoji": ":cloud:", "fields": [{"title": "ENV", "value": "'"$CF_SPACE"'", "short": true}, {"title": "SHA1", "value": "'"<https://github.com/bitzesty/trade-tariff-frontend/compare/$GIT_OLD_REVISION...$GIT_REVISION|$GIT_REVISION>"'", "short": true}]}' \
 $SLACK_WEBHOOK
+
+# Notify GDS about new release
+curl -X POST -H 'Content-type: application/json' \
+-H 'Authorization: Bearer '"$GDS_TOKEN"'' \
+--data '{"repo": "'"$CF_APP"'", "deployment[version]": "'"$GIT_REVISION"'", "deployment[environment]": "'"$CF_APP"'"}' \
```

Comment:
we only need this for staging/prod.

Any they call their staging/prod, integration/production.. not sure if that matters.